tests/basetest.rb

```diff
@@ -133,7 +133,6 @@ def setup
   def teardown
     @device.close unless @device.nil?
     @device = nil
-    GC.start
```

Comment:
It seems like an unnecessary line, but I'd like to know if its removal has an impact on our memory usage first.